Command/GenerateDoctrineCrudCommand.php

```diff
@@ -261,6 +263,37 @@ protected function updateRouting(QuestionHelper $questionHelper, InputInterface
                 '',
             );
         }
+
+        // second, import the bundle's routing.yml file from the application's routing.yml file
+        $routing = new RoutingManipulator($this->getContainer()->getParameter('kernel.root_dir').'/config/routing.yml');
+        try {
+            $ret = $auto ? $routing->addResource($bundle->getName(), $format) : false;
+        } catch (\RuntimeException $e) {
+            // the bundle is already imported form app's routing.yml file
+            $errorMessage = sprintf(
+                "\n\n[ERROR] The bundle's \"Resources/config/routing.yml\" file cannot be imported\n".
+                "from \"app/config/routing.yml\" because the \"%s\" bundle is\n".
+                "already imported. Make sure you are not using two different\n".
+                "configuration/routing formats in the same bundle because it won't work.\n",
+                $bundle->getName()
+            );
```

Comment:
Yes, because the check made by RoutingManipulator is very simple: https://github.com/sensiolabs/SensioGeneratorBundle/blob/master/Manipulator/RoutingManipulator.php#L55-L60